java/core/page-factory-utils/src/main/java/ru/sbtqa/tag/pagefactory/util/PageFactoryUtils.java

```diff
@@ -1,23 +1,25 @@
 package ru.sbtqa.tag.pagefactory.util;
 
+import cucumber.api.Scenario;
 import java.lang.annotation.Annotation;
 import java.lang.reflect.Field;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Locale;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Locale'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)